### ISSUE TITLE
pyric.error [Errno 132] Operation not possible due to RF-kill

### ISSUE THREAD

**Issue by pulsar140** [wifiphisher/wifiphisher#592]
Before opening a new issue, please make sure that what you are about to report has not been reported already. If not, use the following template for reporting:

**Version**: Use `git show | grep commit` command and paste here the output.

**Distribution**: On which GNU/Linux distribution are you using the tool? (e.g. Kali Linux, Gentoo, Debian)
If you know the bug happens on all distributions please mention it. Otherwise, let us know the distribution that you found the bug on.

**Summary**: Describe the bug in a short paragraph.

**Description**: Give a more analytical description here. If the bug you are about to report has to do with the WiFi jamming module, please specify the network cards you are using. If the bug has to do with the rogue Access Point (eg NAT or IP addressing issues), also post the output of the `ifconfig -a` and `iwconfig` commands. Finally, if the bug is about the phishing pages or our HTTP minimal server, please specify the browser you are using to connect to it.

**Steps to Reproduce**: The minimal set of steps necessary to trigger the bug.

**Expected VS actual behavior**: How were you expecting the tool to behave and what actually happened?

**Script output**: Paste the output of the tool, including any stack traces.

**Comment by blackHatMonkey** [comment]
@pulsar140 Thanks for filling a bug report. Does this happen every time you run `wifiphisher` or it is just sometimes?

The `Hard blocked` doesn't always mean that a physical button is blocking it. Sometimes if a card gets stuck it will show as `Hard blocked` even though there is no physical button. Unplugging it usually fixes the issue.

**Comment by pulsar140** [comment]
@blackHatMonkey Yes, it happens every time if I run wifiphisher.
It's an integrated network card - Intel 82579LM.

**Comment by anakin1028** [comment]
I have this experience too, but I don't  know what the root cause is. What I do is I shut down the laptop, and reboot it and  use networkmanager to connect again. Then I disable the network manager and the wifiphisher can run again.

**Comment by SkiddieTech** [comment]
Type
            rfkill unblock all
should do it.

**Comment by pulsar140** [comment]
@SkiddieTech  I tried it, it does not help.

**Comment by blackHatMonkey** [comment]
@pulsar140 Can you try it with the latest `Github` version and see if there is any change?

**Comment by tomtom1021** [comment]
Having an identical problem here after using all suggested fixes with no luck. newest github version installed

**Comment by tomtom1021** [comment]
using alfa awus036nh and nha as my nics

**Comment by anakin1028** [comment]
Do the following operations work for you?
```
ifconfig wlan0 up
iw dev wlan0 scan
```

**Comment by tomtom1021** [comment]
both commands are functional on both nics

**Comment by blackHatMonkey** [comment]
@tomtom1021 I'm changing this issue to a bug.
@anakin1028 Perhaps we need to check for the error in the interfaces whenever we make a call to `pyric`.

**Comment by minhducsun2002** [comment]
Okay, same issue here. However, beside breaking my network card (in software) (Qualcomm Atheros AR9285, bundled with AR5B195), a number of other operations on my PC also stop, including `sudo` being unresponsive (and maybe also `zsh` as Ctrl+C / Ctrl+Z doesn't terminate when `sudo` becomes so). Until I reboot.
